readme.md

```diff
@@ -10,553 +19,44 @@ authentication and more to any API in minutes. Learn more at
-| -------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ------------------- |
-| [`memory-store`](source/memory-store.ts)                                                     | _(default)_ Simple in-memory option. Does not share state when app has multiple processes or servers.                                         | Modern as of v6.0.0 |
-| [`cluster-memory-store`](https://npmjs.com/package/@express-rate-limit/cluster-memory-store) | A memory-store wrapper that shares state across all processes on a server via the [node:cluster](https://nodejs.org/api/cluster.html) module. | Modern              |
-| [`rate-limit-redis`](https://npmjs.com/package/rate-limit-redis)                             | A [Redis](http://redis.io/)-backed store, more suitable for large or demanding deployments.                                                   | Modern as of v3.0.0 |
-| [`rate-limit-memcached`](https://npmjs.org/package/rate-limit-memcached)                     | A [Memcached](https://memcached.org/)-backed store.                                                                                           | Modern as of v1.0.0 |
-| [`rate-limit-mongo`](https://www.npmjs.com/package/rate-limit-mongo)                         | A [MongoDB](https://www.mongodb.com/)-backed store.                                                                                           | Legacy              |
-| [`precise-memory-rate-limit`](https://www.npmjs.com/package/precise-memory-rate-limit)       | A memory store similar to the built-in one, except that it stores a distinct timestamp for each key.                                          | Modern as of v2.0.0 |
-| [`rate-limit-postgresql`](https://www.npmjs.com/package/@acpr/rate-limit-postgresql)         | A [PostgreSQL](https://www.postgresql.org/)-backed store.                                                                                     | Modern as of v1.1.0 |
-
-Take a look at
-[this guide](https://github.com/express-rate-limit/express-rate-limit/wiki/Creating-Your-Own-Store)
-if you wish to create your own store.
-
-## Request API
-
-A `req.rateLimit` property is added to all requests with the `limit`, `used`,
-and `remaining` number of requests and, if the store provides it, a `resetTime`
-Date object. These may be used in your application code to take additional
-actions or inform the user of their status.
-
-Note that `used` includes the current request, so it should always be > 0.
-
-The property name can be configured with the configuration option
-`requestPropertyName`.
-
-## Instance API
+A more in-depth documentation is available on-line at
+[express-rate-limit.mintlify.app](https://express-rate-limit.mintlify.app/overview).
```

Comment:
Right now this looks a little redundant:
<img width="737" alt="Screenshot 2023-10-20 at 11 02 22 AM" src="https://github.com/express-rate-limit/express-rate-limit/assets/114976/ab40612d-acfa-4dc4-90a7-8e0915fd48ce">

Lets change the first one to omit the URL from the text:
```suggestion
[Full documentation](https://express-rate-limit.mintlify.app/overview) is available on-line.
```

Reply:
Maybe we should move this to just under usage, replacing the "An example of its use is as follows:" text?

Reply:
Done